examples/3d/fog_volumes.rs

```diff
@@ -30,20 +30,17 @@ fn main() {
 /// Spawns all the objects in the scene.
 fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
     // Spawn a fog volume with a voxelized version of the Stanford bunny.
-    commands
-        .spawn(SpatialBundle {
-            visibility: Visibility::Visible,
-            transform: Transform::from_xyz(0.0, 0.5, 0.0),
-            ..default()
-        })
-        .insert(FogVolume {
+    commands.spawn((
+        Transform::from_xyz(0.0, 0.5, 0.0),
+        FogVolume {
```

Comment:
This should also have `Visibility::Visible` right? I assume `FogVolume` would be requiring `Visibility::default()` if not. It might not matter though...

Reply:
Same as above, in this case `FogVolume` requires `Visibility` which means it defaults to `Inherited`, and functions as visible.

Reply:
Thanks for addressing my comments!